dist/vl-select.js

```diff
@@ -17,7 +17,6 @@ Promise.all([
 * @property {boolean} data-vl-block - Attribuut wordt gebruikt om ervoor te zorgen dat de textarea getoond wordt als een block element en bijgevolg de breedte van de parent zal aannemen.
 * @property {boolean} data-vl-error - Attribuut wordt gebruikt om aan te duiden dat het select element verplicht is of ongeldige tekst bevat.
 * @property {boolean} data-vl-success - Attribuut wordt gebruikt om aan te duiden dat het select element correct werd ingevuld.
-* @property {boolean} data-vl-disabled - Attribuut wordt gebruikt om te voorkomen dat de gebruiker iets kan kiezen uit het select element.
```

Comment:
was er hier nog iets niet doorgevoerd op master, want ik zie geen change in de src?

Reply:
@Pieter-Beckers heeft deze beantwoord in (comment hieronder): 
https://github.com/milieuinfo/webcomponent-vl-ui-select/pull/166#issuecomment-885443180

"Ik had gewoon een release:prepare uitgevoerd."

Nu ik vermoed dat deze nog niet op master zat ofzo . Nu het is ietswat vreemd dat er een change voorkomt wanneer je daar niets aan aanpast. En dat die ook niet op master zit blijkbaar!
Ik ben even in GIT history gaan kijken en Tom Coemans heeft deze verwijdert op 15/06/2021 (rev. 230188cf02089bb9fc5bce0c36ec0b3e313e84f5).

@LucasSledsens kan je tijdens de merge met master eens bekijken of dit nog voorkomt. Anders zit er mogelijks ergens een probleem?